test-suite/unit-tests/src/utest.mli

```diff
@@ -10,3 +10,6 @@ val mk_bool_test : string -> string -> bool -> OUnit.test
    containing the tests; use __FILE__ for that purpose.
  *)
 val run_tests : string -> OUnit.test list -> unit
+
+(** output channel for the test log file *)
+val log_out_ch : out_channel option ref
```

Comment:
Remove the reference and make the channel a parameter.

Reply:
Well, then how does the test code get the channel without changing the OUnit interface?  See Proof_diffs_test.ml 6-17.

Reply:
The idea is to make it possible for tests to write to the log file for debugging tests: 1) if you have multiple tests in one file, it can be very hard to the output with the right test.  It's obvious when they're in the log file. 2) when a test fails, the expected and actual values show up in the log file.